src/inverseRobot.js

```diff
@@ -7,7 +7,22 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const repairedRobot = {};
+  const arr = [];
+
+  for (const key in robot) {
```

Comment:
maybe you can use more descriptive names instead of just arr and key (checklist pt.3)